.github/workflows/github-actions.yml

```
@@ -67,9 +57,18 @@ jobs:
+            # 서버에 .env 파일 덮어쓰기
+            mkdir -p /home/ubuntu/interview-be
+            cd /home/ubuntu/interview-be
+            cat > .env <<'EOF' # 멀티라인 환경변수 입력으로 사용
+            ${{ secrets.ENV_FILE }}
+            EOF
+            chmod 600 .env
+            
             docker stop interview-be-app || true # 컨테이너가 실행 중이면 중지
             docker rm interview-be-app || true # 기존 컨테이너 삭제
             docker rmi ${{ secrets.DOCKER_USERNAME }}/interview-be-service || true # 기존 이미지 삭제
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)